src/connection_string.ts

```diff
@@ -605,6 +605,7 @@ function setOption(
       if (values[0] == null) {
         break;
       }
+      // eslint-disable-next-line @typescript-eslint/no-base-to-string
```

Comment:
suggest: add a comment explaining why we want to ignore this lint warning

Reply:
Added comment.